src/tokens/TokenImport.tsx

```diff
@@ -95,9 +141,11 @@ export default function TokenImportScreen(_: Props) {
             label={t('tokenImport.input.tokenSymbol')}
             value={tokenSymbol}
             onChangeText={setTokenSymbol}
-            editable={isValidAddress(tokenAddress)}
+            editable={addressState === AddressState.Valid}
```

Comment:
I think this shouldn't be editable at all, right? We should use the `tokenSymbol` that comes from the blockchain data, so the user shouldn't fill the field.